src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -93,6 +96,7 @@ data WalletStorage = WalletStorage
     , _wsReadyUpdates :: [CUpdateInfo]
     , _wsTxHistory    :: !(HashMap (CId Wal) TransactionHistory)
     , _wsHistoryCache :: !(HashMap (CId Wal) (HeaderHash, Utxo, [TxHistoryEntry]))
+    , _wsChangeAddresses :: CAddresses
```

Comment:
Please, make them aligned (and in other places as well). Most of us have `stylish-haskell` applied automatically on save which does this, I recommend )